ansible/files/admin_api_scripts/mount-volume.sh

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+DEVICE=${1:-}
+MOUNT_POINT=${2:-}
+
+if [[ -z "$DEVICE" || -z "$MOUNT_POINT" ]]; then
+  echo "Usage: $0 <device> <mount_point>"
+  echo "Example: sudo ./mount-volume.sh /dev/nvme1n1 /data/150008"
+  exit 1
+fi
+
+#  Mount a block device to a specified mount point
+#  If the device is not formatted, format it as ext4
+#  Set ownership to postgres:postgres and permissions to 750
+#  Add the mount entry to /etc/fstab for persistence across reboots
+
+OWNER="postgres:postgres"
+PERMISSIONS="750"
+FSTYPE="ext4"
+MOUNT_OPTS="defaults"
+FSTAB_FILE="/etc/fstab"
+
+if [ ! -b "$DEVICE" ]; then
+  echo "Error: Block device '$DEVICE' does not exist."
+  exit 2
+fi
+
+if ! blkid "$DEVICE" >/dev/null 2>&1; then
+  echo "Device $DEVICE appears unformatted. Formatting as $FSTYPE..."
+  mkfs."$FSTYPE" -F "$DEVICE"
+else
+  echo "$DEVICE already has a filesystem — skipping format."
+fi
+
```

Comment:
Can we also add a [e2fsck call](https://github.com/supabase/postgres/blob/SUCCESS165_mount_unmount/ansible/files/admin_api_scripts/pg_upgrade_scripts/initiate.sh#L361) here, to make sure the filesystem is healthy before mounting?